### PR TITLE
LPD-19196 Show scheduled document on search on management toolbar

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**
When searching on the management toolbar of the DM page if we search the document scheduled should be returned

1. FF LPD-10701 should be active
2. Create a document with title "test"
3. Create a scheduled document with title "test 2"
4. go to DM page 
5. search by "test"

**Expected**:
The 2 documents should appear

https://github.com/liferay-content-management/liferay-portal/assets/47524751/1d577912-8529-4687-9973-f4c47893b77b

- **LPD-19196 use injection instead of utils**
- **LPD-19196 add test to check that a recently created scheduled file is indexed and showing on the search**
- **LPD-19196 move test to another file to check more cases**
- **LPD-19196 add more tests to check behavior**
- **LPD-19196 reindex if scheduled too**
- **LPD-19196 find fileEntries by several statuses**
- **LPD-19196 buildService**
- **LPD-19196 do not create a draft assetEntry if there is an existing one**
- **LPD-19196 update the fileEntry on the case that is scheduled and the status has changed**
- **LPD-19196 update the asset if the new status is scheduled too and delete the draft if exists**

### CODE REVIEW

File: portal-impl/src/com/liferay/portlet/documentlibrary/service/impl/DLAppHelperLocalServiceImpl.java [hunk 1]
```diff
@@ -883,6 +879,31 @@ protected <T extends RepositoryModel<T>> void triggerRepositoryEvent(
 		}
 	}
 
+	private boolean _addDraftAssetEntry(DLFileVersion dlFileVersion) {
```
Comment (adolfopa) [inline portal-impl/src/com/liferay/portlet/documentlibrary/service/impl/DLAppHelperLocalServiceImpl.java:883]: Shouldn't this be names `_isAddDraftAssetEntry`? The method is not adding anything.

**Review by adolfopa — CHANGES_REQUESTED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
hi @adolfopa 
I added 2 commits with the asked change name.
Thanks!

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Comment by AliciaGarciaGarcia** [comment]
ci:report:2562725094

**Comment by AliciaGarciaGarcia** [comment]
ci:report:2562773814

**Comment by AliciaGarciaGarcia** [comment]
ci:report:2562778927

**Comment by AliciaGarciaGarcia** [comment]
ci:test:content-management

**Comment by AliciaGarciaGarcia** [comment]
rechecking ci:test:content-management because too many unique cases and I do not see correlation but I do not trust none

**Comment by AliciaGarciaGarcia** [comment]
ci:report:2562878968

**Comment by AliciaGarciaGarcia** [comment]
ci:forward:force

**Pull request closed without merging**